app/src/main/java/org/oppia/android/app/help/thirdparty/LicenseTextViewerActivityPresenter.kt

```diff
@@ -0,0 +1,59 @@
+package org.oppia.android.app.help.thirdparty
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import org.oppia.android.R
+import org.oppia.android.app.activity.ActivityScope
+import org.oppia.android.databinding.LicenseTextViewerActivityBinding
+import javax.inject.Inject
+
+/** The presenter for [LicenseTextViewerActivity]. */
+@ActivityScope
+class LicenseTextViewerActivityPresenter @Inject constructor(
+  private val activity: AppCompatActivity
+) {
+
+  /** Handles onCreate() method of the [LicenseTextViewerActivity]. */
+  fun handleOnCreate(dependencyIndex: Int, licenseIndex: Int) {
+    val binding =
+      DataBindingUtil.setContentView<LicenseTextViewerActivityBinding>(
+        activity,
+        R.layout.license_text_viewer_activity
+      )
+    binding.apply {
+      lifecycleOwner = activity
+    }
+    val dependenciesWithLicenseNames =
+      activity.resources.obtainTypedArray(R.array.third_party_dependency_license_names_array)
+    val licenseNamesArrayResId = dependenciesWithLicenseNames.getResourceId(
+      dependencyIndex,
+      -1
```

Comment:
This should be `0` ? Similar to other files.

Reply:
Done.